src/main/java/com/cinema/controllers/OrderController.java

```diff
@@ -0,0 +1,46 @@
+package com.cinema.controllers;
+
+import com.cinema.model.ShoppingCart;
+import com.cinema.model.dto.OrderResponseDto;
+import com.cinema.service.OrderService;
+import com.cinema.service.ShoppingCartService;
+import com.cinema.service.UserService;
+import com.cinema.service.mapper.OrderMapper;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderMapper orderMapper;
+    private final OrderService orderService;
+    private final ShoppingCartService shoppingCartService;
+    private final UserService userService;
+
+    public OrderController(OrderMapper orderMapper, OrderService orderService,
```

Comment:
let's use the annotation above the constructor everywhere

Reply:
done